app/models/messages/UnloadingRemarksRequest.scala

```diff
@@ -36,18 +36,17 @@ case class UnloadingRemarksRequest(
 
 object UnloadingRemarksRequest {
 
-  val transportIdentityLength    = 27
-  val transportIdentityCountry   = 2
```

Comment:
Did this line need to be taken out? val transportIdentityCountry = 2?

Just double checking

Reply:
In main it existed but wasn't used anywhere.